specs/protocol/isthmus/exec-engine.md

```diff
@@ -123,7 +123,8 @@ even if a `withdrawalsRoot` is present and a MPT root is present in the header,
 Any implementation that calculates output root should be careful not to use the header `withdrawalsRoot`.
 
 After Isthmus activation, if there was never any withdrawal contract storage, a MPT root of an empty list
-can be set as the `withdrawalsRoot`
+can be set as the `withdrawalsRoot`. Note that by default there _is_ nonzero storage in
+the [`L2ToL1MessagePasser`][l2-to-l1-mp] at genesis, so the `withdrawalsRoot` will not be the MPT root of an empty list.
```

Comment:
Can't we strengthen this wording to say that it's impossible to have an empty storage root because the `L2ToL1MessagePasser` is guaranteed to have storage because it stores the proxy owner and the implementation address? This is not optional, it is guaranteed by upgrade transactions that are executed as part of consensus.

Reply:
Yes I updated the wording, PTAL. I don't know that it has to do with upgrade transactions though?